frontend/components/markup-modal.js

```diff
@@ -160,29 +237,8 @@ export default class MarkupModal extends React.Component {
               </select>
             </label>
           </p>
-          <p>
-            Markdown&nbsp;
-            <ClickToSelect>
-              <input className="code clickable" readOnly value={markdown} />
-            </ClickToSelect>
-          </p>
-          <p>
-            reStructuredText&nbsp;
-            <ClickToSelect>
-              <input
-                className="code clickable"
-                readOnly
-                value={reStructuredText}
-              />
-            </ClickToSelect>
-          </p>
-          <p>
-            AsciiDoc&nbsp;
-            <ClickToSelect>
-              <input className="code clickable" readOnly value={asciiDoc} />
-            </ClickToSelect>
-          </p>
-          {this.renderDocumentation()}
+          {this.isOpen && this.renderMarkup()}
+          {this.isOpen && this.renderDocumentation()}
```

Comment:
It looks like you've added code to render the documentation but if I look at some examples on https://shields-staging-pr-2441.herokuapp.com/ which include a documentation key, it isn't rendering

Reply:
Fixed in 1a8feaf. Ironically enough, had I specced out the input as you suggested in https://github.com/badges/shields/pull/2441#discussion_r240007528, that would have surfaced the problem.